source/draw/gpu/opengl/shaders/default.vert

```diff
@@ -1,10 +1,10 @@
 #version 300 es
 precision highp float;
-layout(location = 0) in vec2 vertexPosition;
+layout(location = 0) in vec3 vertexPosition;
 layout(location = 1) in vec2 textureCoordinate;
 out vec2 fragmentTextureCoordinate;
 void main() {
-	vec4 position = vec4(vertexPosition.x, vertexPosition.y, 0, 1);
+	vec4 position = vec4(vertexPosition, 1);
```

Comment:
Vector constructors can take complex arguments as multiple arguments to save space.